### ISSUE TITLE
Stringers in Cells should be optional

### ISSUE THREAD

**Issue by GoogleCodeExporter** [DLR-LY/CPACS#375]
```
The stringer definition within the cells of component segments should be 
optional...
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 19 Aug 2013 at 2:07

**Comment by codingpoets** [comment]
Issue does not seem to be solved in the current CPACS version 2.3.1.

**Comment by codingpoets** [comment]
Fixed by 29bf2dfc7a911a72cfaa577309d0e9432a0d7031
